src/main/java/com/openlattice/chronicle/services/download/ParticipantDataIterable.kt

```diff
@@ -23,17 +23,21 @@ import java.util.function.Supplier
 /**
  * @author alfoncenzioka &lt;alfonce@openlattice.com&gt;
  */
-class ParticipantDataIterable(private val supplier: NeighborPageSupplier) : Iterable<Map<String, Set<Any>>> {
+class ParticipantDataIterable(private val columnTitles :List<String>, private val supplier: NeighborPageSupplier) : Iterable<Map<String, Set<Any>>> {
```

Comment:
also minor, but just for style awareness, the kotlin pattern is to do 
```
variableName: Type
```
and not 
```
variableName :Type
```
Definitely confusing going back and forth with flow though, to be sure. The autoformatter should fix this though!